internal/middleware/auth.go

```diff
@@ -27,11 +28,18 @@ func AuthenticateViaCookie(next http.Handler, jimm jujuapi.JIMM) http.Handler {
 	})
 }
 
-// AuthenticateRebac is a layer on top of AuthenticateViaCookie
-// It places the OpenFGA user for the session identity inside the request's context
-// and verifies that the user is a JIMM admin.
-func AuthenticateRebac(next http.Handler, jimm jujuapi.JIMM) http.Handler {
-	return AuthenticateViaCookie(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+// Set of ReBAC Admin API endpoints that do not require authentication.
+var unauthenticatedEndpoints = map[string]any{
```

Comment:
the nittiest of nit-picks: I would prefer struct{} as the value type, because any requires some memory for the type information. It's not like we have a hot path here, so feel free to keep it this way

Reply:
Thanks. That's actually a nice idea. Will do.

Reply:
wouldn't map[string]bool be the most "efficient"?

Reply:
I don't think so. Since the `struct{}` requires no memory space, that would be the most efficient (in terms of size, of course). I couldn't find an official guideline/post on this, but this subreddit is interesting:
https://www.reddit.com/r/golang/comments/qzha6o/for_sets_maptstruct_vs_maptbool/